src/mono/mono/metadata/object.h

```diff
@@ -301,8 +301,6 @@ mono_runtime_exec_main	    (MonoMethod *method, MonoArray *args,
 MONO_API MONO_RT_EXTERNAL_ONLY int
 mono_runtime_set_main_args  (int argc, char* argv[]);
 
-/* The following functions won't be available with mono was configured with remoting disabled. */
-/*#ifndef DISABLE_REMOTING */
```

Comment:
We should keep some variant of this comment: otherwise it's not obvious why there's an apparent API break